howto/manage/setup-tls-for-appliance.md

```diff
@@ -0,0 +1,81 @@
+The Anbox Cloud Appliance uses the self-signed certificate to provide HTTPS services hence it is considered to be used for development or testing purposes in the majority of cases due to the security risks. In case you want to use it for production, you will need to generate a valid SSL certificate and install it to the Anbox Cloud Appliance.
+
+[note type="information" status="Note"]This tutorial assumes you have Anbox Cloud Appliance installed. If you haven't, please follow the [instruction](https://discourse.ubuntu.com/t/install-the-anbox-cloud-appliance/22681) to do so.[/note]
+
+## Add a DNS Record
+
+Setting up DNS redirection depends on your DNS provider. Refer to the documentation of your provider to create a DNS record pointing to the IP/DNS of the AWS instance where the Anbox Cloud appliance is running on.
+
+## Configure the location
+
+Configure the location for the appliance using the created DNS name.
+
+    sudo snap set anbox-cloud-appliance experimental.location=<your DNS name>
```

Comment:
We need to give a warning that this option is experimental and will be removed in the future when a better replacement exists.